.github/workflows/build.yml

```
@@ -0,0 +1,43 @@
+name: CMake Build
+
+on: [pull_request]
+
+env:
+  BUILD_TYPE: Release
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Setup Miniconda
+      uses: conda-incubator/setup-miniconda@v1.7.0
+    
+    - name: Install Dependencies
+      shell: bash
+      run: conda install cmake xtl xproperty nlohmann_json -c conda-forge
+
+    - name: Create Build Environment
+      run: cmake -E make_directory ${{runner.workspace}}/build
+    
+    - name: Check Environment
+      shell: bash
+      run: conda info
+
+    - name: Configure CMake
+      shell: bash
+      working-directory: ${{runner.workspace}}/build
+      # Note the current convention is to use the -S and -B options here to specify source 
+      # and build directories, but this is only available with CMake 3.13 and higher.  
+      # The CMake binaries on the Github Actions machines are (as of this writing) 3.12
+      run: cmake $GITHUB_WORKSPACE -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX -DCMAKE_BUILD_TYPE=$BUILD_TYPE
+
+    - name: Build
+      working-directory: ${{runner.workspace}}/build
+      shell: bash
```

```suggestion
      shell: bash -l {0}
```

You should apply this change everywhere

---

Thanks, on it.

Also, the environment from `conda info` is `None` indeed.

---

Wow, I guess this fixed it. I wonder what it does though.